automation/project/Dependencies.scala

```diff
@@ -3,7 +3,7 @@ import sbt._
 object Dependencies {
   val jacksonV = "2.8.4"
   val akkaV = "2.5.7"
-  val akkaHttpV = "10.0.10"
+  val akkaHttpV = "10.1.0"
```

Comment:
this version bump, as well as the bump to `google-api-services-oauth2` below, are not strictly necessary for this PR. The version bumps do resolve eviction warnings during compile, though.